### DISCUSSION TITLE
Warum erkennt meine frische Installation marine Homematic IP Thermostate nicht?

### DISCUSSION THREAD

**Discussion by ThreeSixty123** [OpenCCU/OpenCCU#2527] [🇩🇪 Fragen&Antworten]
Nachdem meine SD-Karte defekt war und ich meine Backups Wiederhergestellt habe fehlt die RaspberryMatic CCU.

Ich habe alles  zwei mal neu installiert aber ohne erfolg.

der Stick wird erkannt:

Identifying Homematic RF-Hardware: ....HmRF: HMIP-RFUSB/eQ-3 HmIP-RFUSB@usb-0000:01:00.0-1.4, HmIP: HMIP-RFUSB/eQ-3 HmIP-RFUSB@usb-0000:01:00.0-1.4, OK

und die Thermostate sind resette.

Dennoch sehe ich keine neuen Geräte in meinem Postfach.

Egal wie oft ich sie resette und alles (Homeassistant) neustarte.

**Comment by jens-maus** [comment]
Wie hoch ist der angezeigte CarrierSense? Wenn dieser öfters oder gar dauerhaft über 10% liegt ist ein Anlernen bzw. Nutzen von homematicIP Geräten wegen eines Funkstörers nicht möglich.

**Reply by ThreeSixty123** [reply]
Der Carrier Sense ist 0 und Duty Cycle liegt bei 1

**Reply by ThreeSixty123** [reply]
![image](https://github.com/jens-maus/RaspberryMatic/assets/101830341/94fbdbbb-8c92-40dd-b14f-b95bc4684c9d)
 Das kam heute morgen rein,

Ich verstehe aber nicht warum es vorher funktioniert hat?
